File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/PreValidateRedeemBase.java
package com.tokopedia.tokopoints.view.model;

import com.google.gson.annotations.SerializedName;

public class PreValidateRedeemBase {

    @SerializedName("hachikoPreValidateRedeem")
    private PreValidateRedeemBaseValue preValidateRedeem;

    public PreValidateRedeemBaseValue getPreValidateRedeem() {
        return preValidateRedeem;
    }

    public void setPreValidateRedeem(PreValidateRedeemBaseValue preValidateRedeem) {
        this.preValidateRedeem = preValidateRedeem;
    }

    @Override
    public String toString() {
        return "PreValidateRedeemBase{" +
                "preValidateRedeem=" + preValidateRedeem +
                '}';
    }
}

File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/PreValidateRedeemBase.kt
package com.tokopedia.tokopoints.view.model

import com.google.gson.annotations.SerializedName

data class PreValidateRedeemBase(
    @SerializedName("hachikoPreValidateRedeem")
    var preValidateRedeem: PreValidateRedeemBaseValue = PreValidateRedeemBaseValue()
)
